service/user/user_service.py

```diff
@@ -0,0 +1,30 @@
+import os
+import yaml
+import bcrypt
+from flask_httpauth import HTTPBasicAuth
+
+auth = HTTPBasicAuth()
```

Comment:
Willst du hier nicht ne Klasse draus machen? Dann ist das mit der globalen Variable users auch ein bisschen angenehmer

Reply:
Den Impus hatte ich zuerst auch, ich bin mir aber nicht sicher, wie flask_httpauth reagiert, wenn ich ihm eine Klassenmethode statt einer Funktion mit `@auth.verify_password` markiere. Dann müsste sich die Library ja ums Instanziieren der Klasse kümmern.

Wenn wir später mehr Funktionalität für den UserService brauchen, könnte man noch überlegen, daraus eine Klasse zu machen, die dann in einer separaten Funktion `verify_password` genutzt wird. Fürs Erste würde ich aber sagen, dass es so ausreichend ist.

Reply:
`users` habe ich mal in `_users` umbenannt, damit klar wird, dass sie nicht außerhalb des Moduls genutzt werden soll.